core/kubernetes/main/k8s-aws.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+kustomizationFilePath=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
+openhimConsoleVolumePath="${kustomizationFilePath}/openhim/volume/openhim-console/default.json"
+openhimApiUrl=''
+fhirServerUrl=''
+openhimConsoleUrl=''
+
+if [ "$1" == "up" ]; then
+    kubectl apply -k $kustomizationFilePath
+
+    openhimApiUrl=$(kubectl get service openhim-core-service -o=jsonpath={.status.loadBalancer.ingress[0].hostname})
+    fhirServerUrl=$(kubectl get service hapi-fhir-server-service -o=jsonpath={.status.loadBalancer.ingress[0].hostname})
+
+    printf "\n\nOpenHIM Api Url\n---------------\n"$openhimApiUrl"\n"
+    printf "\nHAPI FHIR Url\n--------------\n"$fhirServerUrl"\n\n\n"
+
+    sed -i -E "s/(\"host\": \")\S*(\")/\1${openhimApiUrl}\2/" $openhimConsoleVolumePath
```

Comment:
Maybe add a comment here to say that this is injecting the relevant service URL into the config